App.js

```diff
@@ -1,11 +1,42 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, Button } from 'react-native';
+
+const RULES = [
+  'Give a drink',
+  'Give 2 drinks',
+  'Sociables',
+  'Fuzzy duck',
+  'Story time',
+  'Rule',
+  'Question master',
+  'Thumb master',
+  'Rule',
+  'Never have I ever',
+  'Waterfall',
+  'Girls drink',
```

Comment:
Highly unlikely that the female specimen will be present. Please remove this false positive.